codegen-apollo.yml

```diff
@@ -1,4 +1,7 @@
-schema: ${NEXT_PUBLIC_INDEXER_GRAPHQL_URL}
+schema:
+  - ${NEXT_PUBLIC_HASURA_GRAPHQL_URL}:
+      headers:
+        x-hasura-admin-secret: ${HASURA_GRAPHQL_ADMIN_SECRET}
```

Comment:
Is this public on the web client in the end?

Reply:
I don't understand, but right now queries would go through hasura, however we may end up changing this and have two different clients.